### ISSUE TITLE
Upgrade dependencies to work with Pandas 25

### ISSUE THREAD

**Issue by 1vecera** [scikit-learn-contrib/categorical-encoding#214]
Dear Developers, thanks for this great package.
The issue I ran into:

Running conda install category_encoders gets me version 1.25. (Conda forge is my highest priority channel) 
conda install category_encoders=2.0 forces 2 years old pandas package.
```
The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    category_encoders-2.0.0    |             py_1          41 KB  conda-forge
    mlxtend-0.12.0             |           py36_0         1.2 MB  conda-forge
    pandas-0.22.0              |           py36_1         9.4 MB  conda-forge
    ------------------------------------------------------------
                                           Total:        10.7 MB

The following packages will be UPDATED:

  category_encoders  conda-forge/win-64::category_encoders~ --> conda-forge/noarch::category_encoders-2.0.0-py_1

The following packages will be SUPERSEDED by a higher-priority channel:

  mlxtend            conda-forge/noarch::mlxtend-0.17.0-py~ --> conda-forge/win-64::mlxtend-0.12.0-py36_0

The following packages will be DOWNGRADED:

  pandas                              0.25.1-py36he350917_0 --> 0.22.0-py36_1
```

Both on Windows Server 2012 and CentOs 7.

**Comment by janmotl** [comment]
I have released a new version [2.1](https://pypi.org/project/category-encoders/2.1.0/), which is compatible with Pandas 0.24 and 0.25.

It may take a while until it bubbles into conda-forge.

**Comment by 1vecera** [comment]
Thank you for your answer and the great package! Happy to even learn that main contributer is Czech. ❤
